### ISSUE TITLE
Review / update lpjgpar.json after all updates

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#326]
In f09c408

ec --> dec
et --> det
this needs coordination with the rest of ece2cmor.

Moreover, after all Shaconemo & data request 01.00.28 updates, the list in generate-lpjguess.json.sh actually needs to be updated, however this has become a difficult task since for lpj-guess you inserted all the tables in this list.

**Comment by nierad** [comment]
Sorry, this is still a mystery to me. Do you mean, we can't "remap" ec -> dec, et ->det? 
Ok, sorry, then we will have to remove this from the LPJG output.

**Comment by goord** [comment]
I also don't see a problem there, @treerink can you elaborate?

**Comment by goord** [comment]
Ok, this all sounds very complicated, It's not clear to me how the mapping of the variables can be generated? It contains information that is nowhere else, namely which lpj variables correspond to the requested ones. Therefore any request update requires manual labour, especially if variables get a different meaning...

**Comment by treerink** [comment]
In essence the problem is that "the same" cmor variable in LPJ-GUESS has different LPJ-GUESS code names for different frequencies (tables). This does not match well with the way the data request is designed.

**Comment by nierad** [comment]
So, just to recap: Normally, the drq would determine what output a model will have to produce and in which temporal resolution. With a bit of an effort, we could provide that for LPJG, too. But from the last discussion we had I left with the impression that there is a possibility of a drq for the same variable but with different time-res. Is that true? Because of that it would be impossible to use standard LPJG-output.  
How does it actually look for othe rmodels?

**Comment by klauswyser** [comment]
Some variables are requested at various intervals, e.g. SST or precipitation. For ocean and sea-ice variables XIOS is doing the job nicely: it averages and saves the output at various frequencies. For atmosphere variables the same task i handled by ece2cmor3: IFS saves all variables every 3/6 hours, and then ece2cmor3 computes daily and monthly means when requested.
If I were in your boots,  I wouldn't try to save more variables or the same variable at various intervals at this stage if LPJG isn't prepared for that yet. We can just accept the fact that EC-Earth doesn't save all requested variables (there are plenty of variables from atmopshere and ocean that we will not deliver anyway, so a few more or less doesn't matter in my eyes).

**Comment by treerink** [comment]
In the meanwhile I had a bit closer look, and actually I found no changes in a "part" of the story (save you the details). It means actually that we are fine for the current situation. I updated at the `ece2cmor3` side in 48f2242 the data request changes: et --> det and ec --> dec. Because they where not there at the time of compiling the identified missing list, they miss there now, but as they are in the `lpjguesspar.json` since the last lpjg merge, so that's fine.
